src/css.js

```diff
@@ -46,16 +44,21 @@ export const css = function(key, value) {
     for(prop in styleProps) {
       val = styleProps[prop];
       delete styleProps[prop];
-      styleProps[camelize(prop)] = val;
+      styleProps[dasherize(prop)] = val;
     }
   }
 
   each(this, element => {
     for(prop in styleProps) {
+      let important = undefined;
+      if(typeof styleProps[prop] === 'string' && styleProps[prop].match(/!important$/)) {
+        styleProps[prop] = styleProps[prop].replace(/\s*!important$/, '');
+        important = 'important';
+      }
       if(styleProps[prop] || styleProps[prop] === 0) {
-        element.style[prop] = styleProps[prop];
+        element.style.setProperty(prop, styleProps[prop], important);
```

Comment:
Let's optimize by doing something like this (not tested):

```
const important = /!important$/.test(styleProps[prop]) ? 'important' : undefined;
element.style.setProperty(prop, styleProps[prop].replace(/!important$/, ''), important);
```

Then the logic above it isn't executed if we're removing a property.

Reply:
Thanks, I'll fix it.